library/src/webapp/editor/ckextraplugins/common-wl/js/folder-listing.js

```diff
@@ -155,6 +156,9 @@ $.fn.folderListing = function(options) {
 
 // automatically bind to data-folder-listing divs
 $(document).ready(function() {
-  $('[data-folder-listing]').folderListing();
+  //Check if 'no-highlight' class is present or not and bind accordingly.
+  if (!($('.no-highlight').length > 0)){
+    $('[data-folder-listing]').folderListing();
+  }
 });
```

Comment:
i.e., so that I could, in some other js file, call the plugin with some config, e.g., $('[data-folder-listing]').folderListing('enableHighling':true); or whatever, and it would do all the work for me.